.github/workflows/cleanEnvironments.js

```diff
@@ -1,32 +1,56 @@
 const { Command } = require("commander");
 const fetch = require("node-fetch");
+const core = require("@actions/core");
 
 const program = new Command();
 
 const pathToCloudAPI = "https://staging-cloud.saleor.io/platform/api/";
 
+const snapshotName = "snapshot-automation-tests";
+
+let sendWarningOnSlack = "false";
+let warningMessage = "";
+
 program
   .name("cleanEnvironments")
   .description("Clean environments")
   .option("--token <token>", "token fo login to cloud")
-  .option("--snapshot <snapshot>", "snapshot to revert to")
+  .option(
+    "--environments_to_clean_regex <environments_to_clean_regex>",
```

Comment:
Underscores are unusual for CLIs:

```suggestion
    "--environments-to-clean-regex <environments_to_clean_regex>",
```

Reply:
Not sure why, but it stopped working for me after this change